configs/nri-prometheus-config.yml.sample

```diff
@@ -26,11 +26,11 @@ integrations:
       # It does not include verbose mode. This can lead to a high log volume, use with care.
       audit: false
 
-      # The HTTP client timeout when fetching data from endpoints. Defaults to 30s.
-      # scrape_timeout: "30s"
+      # The HTTP client timeout when fetching data from endpoints. Defaults to 5.
+      # scrape_timeout: "5s"
 
       # Length in time to distribute the scraping from the endpoints.
-      scrape_duration: "5s"
+      scrape_duration: "30s"
```

Comment:
this is not commented out, therefore we are changing the default, is this intended?

Reply:
You are totally right, I saw the `sample` suffix in the file name and I forgot the value can be used as the default (even if it is an example and not the "real" default). I'm restoring the default and including a comment to say that it defaults to that value if it is not set.

Reply:
Updated here: [16b8496](https://github.com/newrelic/nri-prometheus/pull/381/commits/16b8496473292d102f5e35ec3823497a02b15d55)